code/modules/projectiles/guns/magic/wand.dm

```diff
@@ -1,3 +1,6 @@
+//For use in prob()
+#define WAND_WREST_CHANCE (100/121)
```

Comment:
#undef this at the end of the file

Reply:
also, 1/121 * 100 might be more readable?
also, the comment should probably state the reference or tell what the prob() is for

Reply:
Okay, I’ve added the undef, changed the define for readability, and added more to the comment.